auth/template.yaml

```diff
@@ -2010,6 +2015,192 @@ Resources:
         - Key: System
           Value: Authentication
 
+  SharedSignalWafAssociation:
+    Condition: IsNotProduction
+    Type: AWS::WAFv2::WebACLAssociation
+    Properties:
+      ResourceArn: !Sub arn:${AWS::Partition}:apigateway:${AWS::Region}::/restapis/${SharedSignalsApi}/stages/${SharedSignalsApi.Stage}
+      WebACLArn: !GetAtt SharedSignalWAF.Arn
+
+  IPAddressSet:
+    Condition: IsNotProduction
+    Type: AWS::WAFv2::IPSet
+    Properties:
+      Name: !Sub ${AWS::StackName}-shared-signals-ipset
+      Scope: REGIONAL
+      Description: IP address list for Shared Signals API WAF
+      IPAddressVersion: IPV4
+      Addresses:
+        - !Sub "{{resolve:ssm:/${ConfigStackName}/shared-signal/allowed-ipv4-ip-address-list}}"
```

Comment:
How does this resolve if the IP address is a list of IPs? 

For instance, this may work if it resolves a single IP `10....` but doesn't this expect each address to be an entry in this array: 

```
 - IP1
 - IP2
 - ...
```

Instead of:
```
- IP1,IP2,..
```

Reply:
Wouldn't it be better to know in lower environments before pushing to prod/integration?